src/sentry/notifications/notifications/base.py

```diff
@@ -181,29 +178,18 @@ def get_settings_url(self, recipient: Team | User, provider: ExternalProviders)
     def determine_recipients(self) -> Iterable[Team | User]:
         raise NotImplementedError
 
-    def get_notification_providers(self) -> Iterable[ExternalProviders]:
-        # subclass this method to limit notifications to specific providers
-        from sentry.notifications.notify import notification_providers
-
-        return notification_providers()
-
     def get_participants(self) -> Mapping[ExternalProviders, Iterable[Team | User]]:
```

Comment:
@mgaeta why is this method changing in a PR that's supposed to just change a docstring?